docs/interactions/Application_Commands.md

```diff
@@ -48,24 +50,26 @@ Application commands are native ways to interact with apps in the Discord client
-| choices?                   | array of [application command option choice](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-option-choice-structure) | Choices for `STRING`, `INTEGER`, and `NUMBER` types for the user to pick from, max 25                                |
-| options?                   | array of [application command option](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-option-structure)               | If the option is a subcommand or subcommand group type, these nested options will be the parameters                  |
-| channel_types?             | array of [channel types](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types)                                                                               | If the option is a channel type, the channels shown will be restricted to these types                                |
-| min_value?                 | integer for `INTEGER` options, double for `NUMBER` options                                                                                                   | If the option is an `INTEGER` or `NUMBER` type, the minimum value permitted                                          |
-| max_value?                 | integer for `INTEGER` options, double for `NUMBER` options                                                                                                   | If the option is an `INTEGER` or `NUMBER` type, the maximum value permitted                                          |
-| min_length?                | integer                                                                                                                                                      | For option type `STRING`, the minimum allowed length (minimum of `0`, maximum of `6000`)                             |
-| max_length?                | integer                                                                                                                                                      | For option type `STRING`, the maximum allowed length (minimum of `1`, maximum of `6000`)                             |
-| autocomplete? \*           | boolean                                                                                                                                                      | If autocomplete interactions are enabled for this `STRING`, `INTEGER`, or `NUMBER` type option                       |
-
-\* `autocomplete` may not be set to true if `choices` are present.
+| Field                      | Type                                                                                                                                                         | Description                                                                                                          | Valid Types     |
```

Comment:
```suggestion
| Field                      | Type                                                                                                                                                         | Description                                                                                                          | Valid Option Types     |
```
i know this is different than what is in table above, but trying to make explicit